pkg/lib/config/redis.go

```diff
@@ -21,8 +21,10 @@ type RedisConfig struct {
 }
 
 func (c *RedisConfig) SetDefaults() {
+	// Now we use redis pubsub, we need to have much greater number of connections.
+	// https://redis.io/topics/clients#maximum-number-of-clients
 	if c.MaxOpenConnection == nil {
-		c.MaxOpenConnection = newInt(2)
+		c.MaxOpenConnection = newInt(10000)
```

Comment:
I think we should use a hub connection for handling redis subscription, by dynamically issuing `subscribe`/`unsubscribe` commands.

Let's open an issue to improve this later.

Reply:
https://github.com/authgear/authgear-server/issues/822